packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -918,9 +926,14 @@ export default class TradeStore extends BaseStore {
 
     @action.bound
     async accountSwitcherListener() {
+        if (this.root_store.client.landing_company_shortcode === 'maltainvest') {
```

Comment:
standpoint.maltainvest?

Reply:
Could you elaborate what you mean by `standpoint`?

Reply:
There is a new computed property you can leverage here:  checkout `client-store@standpoint`.